app/src/sharedTest/java/org/oppia/android/app/onboarding/OnboardingFragmentTest.kt

```diff
@@ -547,6 +551,37 @@ class OnboardingFragmentTest {
     }
   }
 
+  @Test
+  fun testOnboardingFragment_moveToSlide2_bottomDots_hasCorrectContentDescription() {
```

Comment:
Add this test for `Slide1` too so that we know that it does not start with `1`.

In general developers sometimes make mistake in `0` and `1` as starting point so always make a habit to test the first value, in this case it is `1`.

Reply:
Done